jekyll/_cci2/deployment-integrations.md

```diff
@@ -343,57 +465,75 @@ In the following example, if `build-job` passes and the current branch was the m
 ```
 
 The deployment script pushes the newly created Docker image out to the registry, then updates the K8s deployment to use the
-new image with a `gcloud` command to handle authentication and push the image all at
-once:
+new image with a `gcloud` command to handle authentication and push the image all at once:
 
 ```
 sudo /opt/google-cloud-sdk/bin/gcloud docker -- push us.gcr.io/${PROJECT_NAME}/hello
 ```
 
-The new image is now available in GCR for the GCP infrastructure to
-access. Then, change permissions:
+The new image is now available in GCR for the GCP infrastructure to access. Then, change permissions:
 
 ```
 sudo chown -R ubuntu:ubuntu /home/ubuntu/.kube
 ```
 
-Finally, utilize the patch subcommand of `kubectl` to find the line that specifies the image to use for our container,
-and replaces it with the image tag of the image just built. The K8s deployment
-then intelligently upgrades the cluster by shutting down old containers and
-starting up-to-date ones.
+Finally, utilize the patch subcommand of `kubectl` to find the line that specifies the image to use for our container, and replaces it with the image tag of the image just built. The K8s deployment then intelligently upgrades the cluster by shutting down old containers and starting up-to-date ones.
 
 ```
 kubectl patch deployment docker-hello-google -p '{"spec":{"template":{"spec":{"containers":[{"name":"docker-hello-google","image":"us.gcr.io/circle-ctl-test/hello:'"$CIRCLE_SHA1"'"}]}}}}'
 ```
 
-The full `deploy.sh` file is available on
-[GitHub](https://github.com/circleci/docker-hello-google/blob/master/deploy.sh).
+The full `deploy.sh` file is available on [GitHub](https://github.com/circleci/docker-hello-google/blob/master/deploy.sh).
 A CircleCI 2.0 Google Cloud deployment example project is also available [here](https://github.com/CircleCI-Public/circleci-demo-k8s-gcp-hello-app).
 
+### Google Cloud Orb Example
+
+If you would like to simplify your configuration workflows using a CircleCI orb (a package of configurations that you can use that includes job, commands and executors), the [CircleCI Orbs Registry](https://circleci.com/orbs/registry/) contains several different orb examples for Google Cloud, including the example shown below.
+
+```
+version: 2.1
+
+description: |
+  Install and configure the Google Cloud CLI (gcloud)
+examples:
+  simple_install_and_configure:
+    description: Install the gcloud CLI, if not available
+    usage:
+      version: 2.1
```

Comment:
remove lines 495-502 (this leaves just the part that our user would add to their config)

Reply:
Removed